lib/flow/index.js

```diff
@@ -35,7 +35,12 @@ const getNextSteps = (currentStepId) => {
   return get(flow, currentStepId, []);
 };
 
+const getAllSteps = () => {
+  return values(status);
```

Comment:
FYI `Object.values` will do this without need for lodash. Not worth changing now, but worth being aware of.